### PR TITLE
Create `drush uli` command

### PR DESCRIPTION
**Pull request by mglaman**

### CODE REVIEW

File: custom-commands/drush-uli/uli [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+## Description: Log into the Drupal site with a one-time login link.
+URL=$(ddev exec drush uli | tr -d '\r')
+
+case $OSTYPE in
```
Comment (mglaman) [inline custom-commands/drush-uli/uli:5]: The `launch` host command always prepends the DDEV_URL, so we cannot shortcut to the following

```
ddev launch $(ddev exec drush uli)
```

**Comment by rfay** [comment]
`ddev drush` is already in current alphas. You'll like it.

**Comment by mglaman** [comment]
I know. But that's an extra word. And does `ddev drush uli` launch the browser, though? I thought it didn't due to executing Drush from inside the container

**Comment by rfay** [comment]
`ddev drush uli` outputs the URL. It's the user's job to click it.

**Comment by rfay** [comment]
Nice to have you here!

**Comment by mglaman** [comment]
We can decline this then, I'll keep it for myself :). I've started writing more, but I'll see what is more generic and useful.

**Comment by rfay** [comment]
It would be super cool to do what drush does and pop the browser using terminal characters instead of the "open" or whatever technique `ddev launch` uses.

**Comment by mglaman** [comment]
I've got an idea for the regular Drush command support to do that. I'll open a feature request later

**Pull request closed without merging**